roboteam_ai/test/helpers/WorldHelper.cpp

```diff
@@ -192,4 +192,17 @@ std::pair<roboteam_msgs::World, int> WorldHelper::getWorldMsgWhereRobotHasBall(i
     return std::make_pair(msg, robotWithBallId);
 }
 
+void WorldHelper::resetWorld() {
+    roboteam_msgs::World worldMsg;
+    roboteam_msgs::WorldBall ball;
+    ball.pos.x = 9e9;
+    ball.pos.y = 9e9;
+    ball.vel.x = 9e9;
+    ball.vel.y = 9e9;
+    ball.existence = 99999;
+    ball.visible = true;
+    worldMsg.ball = ball;
+    rtt::ai::world::world->updateWorld(worldMsg);
```

Comment:
This is really ugly. The worldHelper is not build as a controller of the world but rather something that simulates world messages. This also does not properly reset the world as the name implies since there is still history (for example the ball model still gets updated based compared to the previous one. If you want a reset function, I would make it in our AI world, and not do it like this.